### PR TITLE
Develop a React component that will hold the core functionality: a calculator

### PR DESCRIPTION
**Pull request by lenkon**
Project name: Math magicians - components

Project description: In this project, which is the second milestone, the development of the Math Magicians app will be continued. A React component will be developed that will hold the core functionality: a calculator.

#### Project requirements:
In the previous project, you worked on the setup of a React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html), now you should:

- **Delete all the boilerplate** from Create React App (text, images, styles).
- **Create a new directory** called `components`.
- Inside `components`, create a new `Calculator.js` file.
- In `Calculator.js`, create a React component that matches the given design.
- Implement the **design only** (JSX and CSS)
    -  Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use **functional** components
- Create at least **1 child component**
- Once the `Calculator` is ready, import it and **use it in your top-level component**: `App.js`
- **Test it in the browser**; it should render without problems.

_NOTE: Don't add any extra design. Don't add any logic._

Project requirements link: https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    return (
+      <div className="calculator-container">
+        <Total />
+        <div className="calc-buttons">
+          {buttons.map((keyText) => <Button key={keyText.toString()} keyText={keyText} />)}
+        </div>
+      </div>
+    );
+  }
+}
```
Comment (youmari) [inline src/components/Calculator.js]: - [ ] it would be better if the `Caclculator.js` component should be a functional component as required

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @lenkon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (lenkon) [reply src/components/Calculator.js]: Hi @youmari ,

Thanks for your valuable feedback. I have changed the calculator component to a functional one.

Best Regards,
K. Inoulen Shanongba

**Review by prolajumokeoni — APPROVED** [review]
Hi @lenkon 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
  
 - Calculator.js is a functional component 🚀  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**